scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -539,5 +568,5 @@
     @(d/transact conn [{:db/id :rebalancer/config
                         :rebalancer.config/min-utilization-threshold 0.0
                         :rebalancer.config/safe-dru-threshold 0.0
-                        :rebalancer.config/min-dru-diff 0.0000000001
+                        :rebalancer.config/min-dru-diff 0.000000001
```

Comment:
This is inside a comment block, so it does not matter. But will change it back to avoid confusion.